core/domain/app_feedback_report_constants.py

```diff
@@ -65,54 +65,54 @@ class ReportType(enum.Enum):
 # to PascalCase and its values to UPPER_CASE. Because we want to be consistent
 # throughout the codebase according to the coding style guide.
 # https://github.com/oppia/oppia/wiki/Coding-style-guide
-class CATEGORY(enum.Enum): # pylint: disable=invalid-name
+class Category(enum.Enum): # pylint: disable=invalid-name
```

Comment:
Why do you still need all the pylint disables here and below? Wouldn't these changes fix the issues?

Reply:
Removed pylint disables.